src/main/java/seedu/address/logic/commands/ChangePasswordCommand.java

```diff
@@ -0,0 +1,139 @@
+package seedu.address.logic.commands;
+
+import static seedu.address.logic.parser.CliSyntax.PREFIX_CONFIRM_PASSWORD;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NEW_PASSWORD;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_PASSWORD;
+
+import java.nio.charset.StandardCharsets;
+import java.util.logging.Logger;
+
+import com.google.common.hash.Hashing;
+
+import seedu.address.commons.core.LogsCenter;
+import seedu.address.model.UserPrefs;
+
+//@@author charlesgoh
+/**
+ * Changes user's password provided old password is correct and new passwords match.
+ */
+public class ChangePasswordCommand extends Command {
+    public static final String COMMAND_WORD = "changepassword";
+    public static final String COMMAND_ALIAS = "cps";
```

Comment:
Minor: cpw would be more intuitive since people would associate pw rather than ps with password.

Reply:
Personal preference, but yes cpw does sound more intuitive. I suppose i could standardize ps to pw also